plugins/plugin-chart-echarts/src/Timeseries/transformProps.ts

```diff
@@ -169,12 +170,16 @@ export default function transformProps(chartProps: ChartProps): EchartsProps {
     },
     tooltip: {
       ...defaultTooltip,
-      trigger: 'axis',
-      formatter: params => {
+      trigger: richTooltip ? 'axis' : 'item',
+      formatter: (params: any) => {
+        const value: any = !richTooltip ? params.value : params[0].value[0];
```

Comment:
Same comment about the usage of "any" here and in the other places where it has been used